### PR TITLE
Contact form Added: Mobile Version

### PR DESCRIPTION
**Pull request by habitoz**
contact form added according to the design and requirement given so far.
the following files has been altered due to this change
     - index.html is altered to include the contact form and imputs given from the design
     - style.css file is also altered to include styles for the contact form section
     - new image added to the assets folder

### CODE REVIEW

**Review by GedeonTS — COMMENTED** [review]
# INVALID REVIEW REQUEST

Hi @habitoz , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

As This milestone is a [pair-programming activity](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#work-and-submission-mode), Kindly make sure to have contribution from your partner on this project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: styles.css [hunk 1]
```diff
@@ -340,6 +340,68 @@ main {
...
+  padding: 45px 24px;
+  background-position-x: right;
+  background-origin: border-box;
+  background-color: var(--primary-color);
+  border-top-left-radius: 15%;
+}
+
+.contact-me-info {
+  margin-top: 40px;
+  text-align: center;
```
Comment (Shedrack-Sunday) [inline styles.css]: - [ ] Nice work you did there 👍 . However, based on project requirements, you're meant to use Flexbox to style this section. This is to test your skill using that technique, which is an industry best practice and gives you more flexibility 👍 

## Requirement:
image

File: index.html [hunk 2]
```diff
@@ -143,33 +143,56 @@ <h1 class="work-title">Tonic</h1>
...
+                <ul>
+                    <li class="form-control">
+                        <input required placeholder="Your name" name="name" id="name" type="text" class="form-input"/>
+                    </li>
+                    <li class="form-control">
+                        <input required placeholder="Your email" name="email" id="email" type="email" class="form-input"/>
+                    </li>
+                    <li class="form-control">
+                        <textarea required placeholder="Write your message here"  name="message" id="message" cols="20" rows="10" class="form-input"/></textarea>
+                    </li>
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] For this section, `Name` input should have `maxlength="30"` attribute.
- [ ] Message input has `maxlength="500"` attribute.
- These are necessary validations to keep your code professional

## Requirement :
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @habitoz        👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well so far in the project,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights   
- Good use of GitHub features ✔️   
- Nice UI 👍 

  
# Required Changes  🌵 

- [x] You made some impressive reviews in the comments, which I must say is a job well done. However, there are some which missed 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by AbelG101 — APPROVED** [review]
# Hi @habitoz 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Good replication of Figma design ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**